### PR TITLE
build: Upgrade puppeteer and related dependencies

### PR DESCRIPTION
**Pull request by tylerbutler**
Upgrades puppeteer and related dependencies to the latest version.

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -15589,15 +15597,15 @@ packages:
   /@fluid-experimental/attributable-map/2.0.0-internal.8.0.0:
     resolution: {integrity: sha512-fmBMnKDkWeIDtTMlacDGEN7mX7Lpc2Mg3BuRscDGOU51tn1cxIR7WPcWWbn4wcOhCvqmC2UuNRHUEWumpmpWxg==}
     dependencies:
-      '@fluid-internal/client-utils': 2.0.0-internal.8.0.0
-      '@fluidframework/core-interfaces': 2.0.0-internal.8.0.0
-      '@fluidframework/core-utils': 2.0.0-internal.8.0.0
-      '@fluidframework/datastore-definitions': 2.0.0-internal.8.0.0
-      '@fluidframework/driver-utils': 2.0.0-internal.8.0.0
+      '@fluid-internal/client-utils': 2.0.0-internal.8.0.4
```
Comment (jason-ha) [inline pnpm-lock.yaml:15600]: Got the funky pickups back

**Review by jason-ha — CHANGES_REQUESTED** [review]
Need the cleaner pnpm lock changes.
Should the shell change be kept when just going to 19?

File: pnpm-lock.yaml (hunk 1 above)
Comment (tylerbutler) [reply pnpm-lock.yaml:15600]: Ugh, how annoying. Will fix.

**Comment by tylerbutler** [comment]
> Should the shell change be kept when just going to 19?

No, I don't think so. 19 still has the old headless behavior, though it now spams a warning that the default is changing.

File: pnpm-lock.yaml (hunk 1 above)
Comment (tylerbutler) [reply pnpm-lock.yaml]: Should be fixed in latest.

File: package.json [hunk 2]
```diff
@@ -38,7 +38,7 @@
 		"checks:fix": "fluid-build --task checks:fix",
 		"ci:build": "fluid-build --task ci:build",
 		"ci:build:docs": "fluid-build --task ci:build:docs",
-		"ci:test:jest": "npm run test:jest:report",
+		"ci:test:jest": "puppeteer browsers install chrome && npm run test:jest:report",
 		"ci:test:jest:coverage": "c8 --no-clean npm run test:jest:report",
```
Comment (jason-ha) [inline package.json:42]: Will want install for coverage too, I think.

Comment (tylerbutler) [reply package.json]: Done.

**Comment by tylerbutler** [comment]
Superseded by other PRs. Closing.

**Pull request closed without merging**